lib/aggregate.js

```diff
@@ -111,11 +111,11 @@ Aggregate.prototype.model = function(model) {
   this._model = model;
   if (model.schema != null) {
     if (this.options.readPreference == null &&
-        model.schema.options.read != null) {
+      model.schema.options.read != null) {
```

Comment:
For the future, please avoid making these sort of minor style changes. They make it harder to review your code.